### ISSUE TITLE
children raider? 

### ISSUE THREAD

**Issue by PaddyVu** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#208]
All kind of raider always like this,in battle they're small like this too,i dont have any other mod,only this mod 
![image](https://user-images.githubusercontent.com/64102260/79971071-09b91a80-84be-11ea-81c6-a24491f05724.png)

**Comment by Tyler-IN** [comment]
That's freaking weird, but we don't have anything that screws with ages.

**Comment by Tyler-IN** [comment]
Could you attach the save file where this happened so we can analyze it?

**Comment by PaddyVu** [comment]
> Could you attach the save file where this happened so we can analyze it?

i cant attrach the save file,github dont support this kind of file,but it's only happen when i open fix launcher that contain this mod,if i use normal launcher everything is back to normal,this is weird

**Comment by PaddyVu** [comment]
> Could you attach the save file where this happened so we can analyze it?
Ok i think this is not from your mod,i uncheck the mod and use fix launcher so it's still cause this,i should ask dev from that mod 
![image](https://user-images.githubusercontent.com/64102260/80010928-5704ae80-84f5-11ea-8591-c3b89de72a21.png)

**Comment by Tyler-IN** [comment]
You should be able to zip the file and upload it. Worst case scenario you can upload it to some other file sharing site and paste the link here.

**Comment by PaddyVu** [comment]
> You should be able to zip the file and upload it. Worst case scenario you can upload it to some other file sharing site and paste the link here.

Hi again,the problem has been mentioned in previous issue,deleted and reinstall the game with only your mod and bannerlord mod launcher and it's working again,all fault is from "Fixed launcher"
![image](https://user-images.githubusercontent.com/64102260/80083658-0f773480-8580-11ea-931f-e739042d4e69.png)

**Comment by Fumblesneeze** [comment]
> > You should be able to zip the file and upload it. Worst case scenario you can upload it to some other file sharing site and paste the link here.
> 
> Hi again,the problem has been mentioned in previous issue,deleted and reinstall the game with only your mod and bannerlord mod launcher and it's working again,all fault is from "Fixed launcher"

I bet you it's not "Fixed launcher" fault. Probably some mod xml files that weren't deleted properly. You should just delete the Modules folder completely then verify game files on steam to get a clean modules folder.

**Comment by PaddyVu** [comment]
> > > You should be able to zip the file and upload it. Worst case scenario you can upload it to some other file sharing site and paste the link here.
> > 
> > 
> > Hi again,the problem has been mentioned in previous issue,deleted and reinstall the game with only your mod and bannerlord mod launcher and it's working again,all fault is from "Fixed launcher"
> 
> I bet you it's not "Fixed launcher" fault. Sounds like some mod xml files that weren't deleted properly.

I dont think so because when i switching between normal launcher and fixed launcher give different result,while the normal working fine but fixed launcher cause this.
